about.html

```diff
@@ -0,0 +1,126 @@
+        </div>
+
+    </div>
+</header>
+
+<section class="firstPage">
+  <div class="layer">
+
+    <div class="bars">
+      <img src="images/bars2.png" alt="bars icon" id="bars">
+  </div>
+      <div class="headings">
+          <h1>EA Sports Presents</h1>
+          <h2>Fifa 2021 Pre Game Press Conference</h2>
+      </div>
+      <div class="information">
+          <p>FIFA 21 is an association football simulation video game published by Electronic Arts as part of the FIFA series.[1] It is the 28th installment in the FIFA series, and was released on 9 October 2020 for Microsoft Windows, Nintendo Switch, PlayStation 4 and Xbox One. Enhanced versions for the PlayStation 5 and Xbox Series X and Series S were released on 3 December 2020, in addition to a version for Stadia in March 2021.</p>
+      </div>
+      <div class="contact">
+        <p>Please contact us per Email for any further questions about Fifa 21!</p>
+        <p class="mail"><strong>fifa21@gmail.org</strong></p>
+      </div>
+  </div>
+</section>
+
+<section class="coverpage">
+  <h2>Fifa 21 Coverpage Athlete</h2>
+  <hr>
+  <p>From humble AS Bondy to the feted Clairefontaine, Kylian Mbappé was always destined for greatness. The new FIFA 21 cover star has scored FIFA World Cup™-winning goals, won league titles, was named Golden Boy in 2017, and won Ligue 1 Player of The Year at the age of 20. He signed his first professional contract in 2016. A lot can happen in four years when you’ve got the world at your feet.</p>
+  <div class="alternate">
+    <img src="images/img5a.jpg" alt="Mbappe" class="bottom">
+    <img src="images/img3a.jpg" alt="Mbappe" class="top">
+  </div>
+</section>
+
+<section class="games">
+  <h2>See the past Fifa games</h2>
+  <hr>
+  <p>Take a look at the last two Fifa games which EA Sports released in the years 2020 and 2019. </p>
+  <div class="gameImage">
+    <div class="alt">
+      <img src="images/img8.jpg" alt="fifa20" class="bottom">
+      <img src="images/img7.jpg" alt="fifa20" class="top">
+    </div>
+    <div class="alt">
+      <img src="images/img10.jpg" alt="fifa19" class="bottom">
+      <img src="images/img9.jpg" alt="fifa19" class="top">
+    </div>
+  </div>
+</section>
+
+<section class="partner">
+  <h2>Partners</h2>
+  <hr>
+  <div class="partners">
+      <img src="images/premier.png" alt="premier league logo" class="partner1">
+      <img src="images/laliga.png" alt="laliga logo" class="partner2">
+      <img src="images/league1.png" alt="league1 logo" class="partner3">
+      <img src="images/bundes.png" alt="bundesliga logo" class="partner4">
+      <img src="images/champions2.jpg" alt="champions league logo" class="partner6">
+      <img src="images/euro.png" alt="europa logo" class="partner6">
+  </div>
+
+</section>
+
+<footer class="footer">
+  <div>
+      <img src="images/unnamed.png" alt="logo">
+  </div>
+  <div>
+      <p>@2021 Kingstalux creativity</p>
+      <p>Some Rights Reserved</p>
+  </div>
+</footer>
```

Comment:
- [OPTIONAL] I'm gonna mark this as optional because according to the requirement, you follow the request, but consider using this for the footer. As you said, the partners and footer section are not required to be inside the more button, but according to the template, that part is a little confusing so, it's ok if you don't want to change it again. Please, consider this to change on the footer: 🐱 
![image](https://user-images.githubusercontent.com/20563252/132955692-cab9c7e2-c628-4e4e-8987-07eb44a61c60.png)

Reply:
I'll leave this for now.